lectures/commod_price.md

```diff
@@ -347,7 +345,7 @@ The code below implements this iterative process, starting from $p_0 = P$.
 The distribution $\phi$ is set to a shifted Beta distribution (although many
 other choices are possible).
 
-The integral in [](eq:dopf3) is computed via Monte Carlo.
+The integral in [](eq:dopf3) is computed via {ref}`Monte Carlo <Monte-Carlo>`.
```

Comment:
Is `Monte-Carlo` a label or a document? If a document then we should use `{doc}`

```
{doc}`monte_carlo.md`
```